src/frontend/apps/desk/src/features/teams/api/useTeams.tsx

```diff
@@ -2,18 +2,44 @@ import { UseQueryOptions, useQuery } from '@tanstack/react-query';
 
 import { APIList, fetchAPI } from '@/api';
 
-interface TeamResponse {
+interface Access {
+  id: string;
+  role: string;
+  user: string;
+  abilities: {
+    get: boolean;
+    patch: boolean;
+    put: boolean;
+    delete: boolean;
+    manage_accesses: boolean;
+    set_role_to: [];
+  };
```

Comment:
do we need `abilities`? User's role should be enough

Reply:
is it a bad practice to just ignore it in the frontend? Sorry if my question is dumb

Reply:
It is not dumb, of course it is cleaner if the back is synch with the front, but I guess it depend the case as well.
If you have a big endpoint used by different resources, with lot of useless data, to limit what your front need in the TS type make sense.

beb5da628fb16852d5a051442f3fcae0cf40f2e1

Let's remove it from the back if not needed (I will open an issue).